common/src/IO/ZipFileSystem.cpp

```diff
@@ -24,41 +24,80 @@
 #include "IO/DiskFileSystem.h"
 #include "IO/IOUtils.h"
 
+#include <miniz/miniz.h>
+
 #include <cassert>
 #include <cstring>
 #include <memory>
-
-#include <wx/log.h>
-#include <wx/mstream.h>
-#include <wx/zipstrm.h>
+#include <string>
 
 namespace TrenchBroom {
     namespace IO {
-        ZipFileSystem::ZipCompressedFile::ZipCompressedFile(std::shared_ptr<wxZipInputStream> stream, std::unique_ptr<wxZipEntry> entry) :
-        m_stream(std::move(stream)),
-        m_entry(std::move(entry)) {}
+        // MinizArchive
 
-        MappedFile::Ptr ZipFileSystem::ZipCompressedFile::doOpen() const {
-            const auto path = Path(m_entry->GetName().ToStdString());
+        class MinizArchive {
```

Comment:
Maybe leave a comment that this is just a resource management class for the actual mz archive.